packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -91,3 +91,14 @@ export const borderAndBoxShadowStyle = (
     }
   };
 };
+
+/**
+ * @private
+ */
+export const errorBarStyle = (theme: Theme): IStyle => {
```

Comment:
Is this still being used or can it be removed?

Reply:
Yes, please remove this